pipeline/data/importers.py

```diff
@@ -0,0 +1,221 @@
+import shutil
```

Comment:
I generally like to have a short file-level comment explaining why this file exists, e.g.